### ISSUE TITLE
Codicons are not vertically centered

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#92643]
@misolori assigning first to you and feel free to assign back to me.

1. Debug two sessions such that the Debug Session is shown in the CallStack view
2. Hover over a session -> notice how the debug actions are not vertically centered

![Screenshot 2020-03-13 at 13 55 20](https://user-images.githubusercontent.com/1926584/76622706-4c3c1e80-6532-11ea-852c-24d3eb7069fd.png)

Easy way to debug two sessions:
Have the following `test.js` file:
```
console.log('hello');
```

Have the following launch.json and run both configurations:
```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "type": "node",
            "request": "launch",
            "name": "Launch Program",
            "program": "${workspaceFolder}/test.js"
        },
        {
            "type": "node",
            "request": "launch",
            "name": "Launch Program2",
            "program": "${workspaceFolder}/test.js"
        }
    ]
}
```

**Comment by misolori** [comment]
I made a recent a change (https://github.com/microsoft/vscode/commit/ee04d73ca9a83f422925738ca743e221b741623b) that removed the vertical alignment on codicons and used an alternate method via flexbox. However this caused some codicons in list views to be off-centered. I've updated it so that all codicons now use the same method.

Verification steps: verify that codicons are vertically centered in:

1. Debug viewlet w/ multiple debug sessions (see original post)
  ![image](https://user-images.githubusercontent.com/35271042/76719457-1300db80-66f7-11ea-8d15-4fc62a8d7033.png)

2. SCM actions
  ![image](https://user-images.githubusercontent.com/35271042/76719441-09777380-66f7-11ea-90d8-359163ed285f.png)

3. Custom views like Timeline and NPM Scripts
  ![image](https://user-images.githubusercontent.com/35271042/76719475-20b66100-66f7-11ea-94ca-c0aa37a1e7b3.png)
